configuration/redis/redis_test.go

```diff
@@ -118,9 +120,43 @@ func TestConfigurationStore_Get(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "test does not throw error for wrong type during get all",
```

Comment:
The behavior introduced in this PR is - "Whenever a key type is unsupported, we log a warning and skip returning it". Is it being tested here? It looks like we are trying to get all the values and all of them are returned.